src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const standartCost = 40;
+  const rentDays3 = 3;
```

Comment:
```suggestion
  const smallRentTime= 3;
```
or something like that
if this number will change in future - you should change both variable name in all places and value of this variable